seqr/management/commands/lift_project_to_hg38_tests.py

```diff
@@ -0,0 +1,357 @@
+            },
+            'I000006_hg00733': {
+                'ab': 0, 'ad': None, 'gq': 96, 'sampleId': 'HG00733', 'numAlt': 1, 'dp': 42, 'pl': None,
+                'cn': 2, 'end': None, 'start': None, 'numExon': None, 'defragged': None, 'qs': None,
+            },
+            'I000007_na20870': {
+                'ab': 0.70212764, 'ad': None, 'gq': 46, 'sampleId': 'NA20870', 'numAlt': 1, 'dp': 50, 'pl': None,
+                'cn': 2, 'end': None, 'start': None, 'numExon': None, 'defragged': None, 'qs': None,
+            }
+        },
+        'genotypeFilters': '',
+        'genomeVersion': '37',
+        'hgmd': {'accession': None, 'class': None},
+        'liftedOverGenomeVersion': None,
+        'liftedOverChrom': None,
+        'liftedOverPos': None,
+        'mainTranscriptId': TRANSCRIPT_1['transcriptId'],
+        'originalAltAlleles': ['G'],
+        'populations': {
+            'callset': {'an': 32, 'ac': 1, 'hom': None, 'af': 0.031, 'hemi': None, 'filter_af': None},
+            'g1k': {'an': 0, 'ac': 0, 'hom': 0, 'af': 0.0, 'hemi': 0, 'filter_af': None},
+            'gnomad_genomes': {'an': 0, 'ac': 0, 'hom': 0, 'af': 0.0, 'hemi': 0, 'filter_af': None},
+            'exac': {'an': 121336, 'ac': 6, 'hom': 0, 'af': 0.00004942, 'hemi': 0, 'filter_af': 0.000242306760358614},
+            'gnomad_exomes': {'an': 245714, 'ac': 6, 'hom': 0, 'af': 0.000024418633044922146, 'hemi': 0,
+                              'filter_af': 0.00016269686320447742},
+            'topmed': {'an': 0, 'ac': 0, 'hom': 0, 'af': 0.0, 'hemi': 0, 'filter_af': None},
+            'sv_callset': {'an': None, 'ac': None, 'hom': None, 'af': None, 'hemi': None, 'filter_af': None},
+        },
+        'pos': 3339582,
+        'predictions': {
+            'splice_ai': None, 'eigen': None, 'revel': None, 'mut_taster': None, 'fathmm': None, 'polyphen': None,
+            'dann': None, 'sift': None, 'cadd': 17.26, 'metasvm': None, 'primate_ai': None, 'gerp_rs': None,
+            'mpc': None, 'phastcons_100_vert': None, 'strvctvre': None,
+        },
+        'ref': 'GAGA',
+        'rsid': None,
+        'transcripts': {
+            'ENSG00000135953': [TRANSCRIPT_1],
+            'ENSG00000228198': [TRANSCRIPT_2],
+        },
+        'variantId': '2-003339582-GAGA-G',
+        'xpos': 2003339582,
+        'end': None,
+        'svType': None,
+        'numExon': None,
+        '_sort': [2003339582],
+    },
+]
+
+
+class LiftProjectToHg38Test(TestCase):
+    fixtures = ['users', '1kg_project']
+
+    @mock.patch.object(__builtin__, 'raw_input')
+    @mock.patch('logging.getLogger')
+    @mock.patch('seqr.views.utils.dataset_utils.get_elasticsearch_index_samples')
+    @mock.patch('seqr.utils.elasticsearch.utils.get_es_variants_for_variant_tuples')
+    @mock.patch('seqr.utils.elasticsearch.utils.get_single_es_variant')
+    def test_command(self, mock_single_es_variants, mock_get_es_variants, mock_get_es_samples, mock_getLogger, mock_input):
```

Comment:
Can you please add testing for the error cases?

Reply:
Error cases have been added. I inserted new database rows or updated existing rows to generate errors. I'm not sure whether it is the right way to do test for the error cases.